build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
```

Comment:
What does "automatic" mean though? Reading that I think it means automatically after a build which is the opposite direction we are going.